packages/@okta/vuepress-site/docs/reference/api/zones/index.md

```diff
@@ -657,7 +747,9 @@ Updates an existing Network Zone
 
 A valid [Zone object](#zone-object) with the id of the network zone to update is required.
 
-The updated Network Zone type must be the same as the existing type.
+The updated Network Zone type must be the same as the existing type. 
+
+You may update the usage (e.g. `POLICY`, `BLOCKLIST`) of a Network Zone by updating the `usage` attribute.
```

Comment:
we avoid the use of latin abbreviations such as `e.g.`

You may update the usage (for example: `POLICY`, `BLOCKLIST`) of a Network Zone by updating the `usage` attribute.